### PR TITLE
document route53 IAM policy

### PR DESCRIPTION
**Pull request by bobzoller**
### 1. Why is this pull request needed and what does it do?
This PR documents the required IAM permissions to use the route53 plugin.  AFAICT, the alternative is to read the source code and AWS docs to figure it out.

### 2. Which issues (if any) are related?
- #2924 
- #2859
- #2983 

### 3. Which documentation changes (if any) need to be made?
This is _only_ a documentation change!

### 4. Does this introduce a backward incompatible change or deprecation?
No.

### CODE REVIEW

**Comment by miekg** [comment]
fyi: the docs in man/ are generated, so you don't have to include them here. The READM.md is the canonical doc.

**Comment by miekg** [comment]
Is it possible to add a `## Also See` section at the end to link to extra documentation that has a stable URL?

File: man/coredns-route53.7 [hunk 1]
```diff
@@ -19,7 +21,7 @@ The route53 plugin can be used when coredns is deployed on AWS or elsewhere.
 
 .nf
 route53 [ZONE:HOSTED\_ZONE\_ID...] {
-    [aws\_access\_key AWS\_ACCESS\_KEY\_ID AWS\_SECRET\_ACCESS\_KEY]
+    aws\_access\_key [AWS\_ACCESS\_KEY\_ID AWS\_SECRET\_ACCESS\_KEY]
```
Comment (dilyevsky) [inline man/coredns-route53.7:24]: Both `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are required.

**Review by dilyevsky — CHANGES_REQUESTED** [review]

**Comment by miekg** [comment]
are you able to change the PR according to the comments? Otherwise I'll close it.

**Comment by miekg** [comment]
closing this tomorrow - if I can remember to do it.

**Pull request closed without merging**